components/helm-broker/internal/ybundle/populator.go

```diff
@@ -57,31 +57,46 @@ func (b *Populator) Init() error {
 		return err
 	}
 
+	loaded := 0
+	failed := 0
 	for entryName, versions := range idx.Entries {
 		for _, v := range versions {
-			bundleReader, bundleCloser, err := b.repo.BundleReader(entryName, v.Version)
+			err := b.loadBundle(entryName, v.Version)
 			if err != nil {
-				return errors.Wrapf(err, "while reading bundle archive for name [%s] and version [%v]", entryName, v.Version)
+				failed = failed + 1
+				b.log.Warnf("Could not load bundle: %s", err.Error())
+			} else {
+				loaded = loaded + 1
 			}
-			defer bundleCloser()
+		}
+	}
+	b.log.Infof("Loading bundles completed. Successfully loaded %d bundles, failed %d bundles.", loaded, failed)
```

Comment:
Could you write a test which will check if we log this with a correct number of loaded/failed bundles?

Reply:
In my opinion, testing if some info log entry was done is not a good way, like it is described here: https://dzone.com/articles/never-test-logging